src/main/java/uk/gov/hmcts/reform/sscs/hearing/mapping/PartyFlagsMapping.java

```diff
@@ -0,0 +1,145 @@
+package uk.gov.hmcts.reform.sscs.hearing.mapping;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.model.servicehearingvalues.PartyFlags;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.ADJOURN_CASE_INTERPRETER_LANGUAGE;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.DISABLED_ACCESS;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.DWP_PHME;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.DWP_UCB;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.HEARING_LOOP;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.IS_CONFIDENTIAL_CASE;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.SIGN_LANGUAGE_TYPE;
+import static uk.gov.hmcts.reform.sscs.hearing.mapping.PartyFlagsMap.URGENT_CASE;
+
+public class PartyFlagsMapping {
+
+    public List<PartyFlags> getPartyFlags(SscsCaseData caseData) {
+        List<PartyFlags> partyFlagsList = new ArrayList<>();
+        partyFlagsList.add(mapSignLanguageType(caseData));
+        partyFlagsList.add(disabledAccess(caseData));
+        partyFlagsList.add(hearingLoop(caseData));
+        partyFlagsList.add(confidentialCase(caseData));
+        partyFlagsList.add(dwpUcb(caseData));
+        partyFlagsList.add(dwpPhme(caseData));
+        partyFlagsList.add(urgentCase(caseData));
+        partyFlagsList.add(adjournCaseInterpreterLanguage(caseData));
+        return partyFlagsList;
```

Comment:
You can replace all of this by using Arrays.asList

Reply:
Will look a little nicer but will achieve the same...